main.cpp

```diff
@@ -231,6 +243,32 @@ float ElectricGuitar::brightTone (float noteInfo)
     return noteInfo * highshelf;
 }
 
+void ElectricGuitar::tuneString(int currentNoteInfo, int targetNote)
+{
+    if (currentNoteInfo < targetNote)
+    {
+        std::cout << "You were flat! ";
+        while (currentNoteInfo < targetNote)
+        {
+            ++currentNoteInfo;
+        }
+        std::cout << "Now you're in tune." << std::endl;
+    } 
+        else if (currentNoteInfo > targetNote)
+        {
+            std::cout << "You were sharp! ";
+            while (currentNoteInfo > targetNote)
+            {
+                currentNoteInfo -= 1;
+            }
+            std::cout << "Now you're in tune." << std::endl;
+        } 
+            else if (currentNoteInfo == targetNote)
+            {
+                std::cout << "You're already in tune!!" << std::endl;
+            }
+}
+
```

Comment:
This indentation is still not correct.
```
void functionName( parameters )
{
    if( expression )
    {

    }
    else if( expression ) 
    {

    }
}
```
Keywords need to line up vertically, and curly braces indicating blocks associated with those keywords need to line up with those keywords.